drr.go

```diff
@@ -4,20 +4,20 @@ package drr
 
 import (
 	"context"
-	"fmt"
+	"errors"
 	"reflect"
 )
 
 var (
-	// InvalidPriorityValueError error is returned by Input method when
+	// ErrInvalidPriorityValueError error is returned by Input method when
```

Comment:
Thanks for spotting these fixes and taking care of them!

So, at this point I was wondering if "ErrBlaBlaError" is kind of repetitive. I think we can drop the final "Error" string.

Reply:
No worries :)